### PR TITLE
Flexibus/#497/profile page design

### LINKED ISSUE #497: Design Profile Page
New design for Player Profile Page
(https://sport-portal-dev.herokuapp.com/users/XX/profile)

Button "Profil bearbeiten" goes to https://sport-portal-dev.herokuapp.com/users/XX/edit

### PR DESCRIPTION
**Pull request by jonashering**
Closes #497.

Finishes redesign of profile site.

### CODE REVIEW

File: app/views/devise/registrations/show.html.erb [hunk 1]
```diff
@@ -1,35 +1,43 @@
...
+  <div class="row">
+    <% if @user.favourite_sports.present? %>
+        <p class="description-tag"><%= model_class.human_attribute_name(:favourite_sports) %></p>
+        <p class="description-text"><%= @user.favourite_sports %></p>
+    <% end %>
+  </div>
+  <div class="row">
+    <% if can? :edit, @user %>
+        <%= link_to t('.edit', :default => t('navbar.drop-down.edit_profile')),
+                    edit_user_path(@user), :class => 'btn btn-default btn-save-edit-user btn-centered' %>
```
Comment (jkbe) [inline app/views/devise/registrations/show.html.erb]: It seems all properties of `btn-default` are overwritten so it's safe to remove.

File: app/assets/stylesheets/users.css [hunk 2]
```diff
@@ -35,3 +35,39 @@
...
+
+.page-header {
+    margin-bottom: 0;
+}
+
+.btn-centered {
+    margin: 0;
+}
+
+.profile-image {
```
Comment (jkbe) [inline app/assets/stylesheets/users.css]: Why not make this an `id` for a change? 👯

**Review by jkbe — CHANGES_REQUESTED** [review]
Looks good. Just one minor issue and a suggestion.

**Comment by jonashering** [comment]
@jak-ing Last commit addresses changes requested. Please recheck!

**Review by jkbe — APPROVED** [review]

**Pull request merged**